### DISCUSSION TITLE
請問绘制工具组件所绘制的物件,綠框綠线會消失

### DISCUSSION THREAD

**Discussion by stan88** [zouyaoji/vue-cesium#338] [Q&A]
您好,在绘制的物件上的綠框,初始繪制沒問題(圖1),
但是按下[绘制面]後,綠框同時消失(圖2)
後續在绘制時也無綠线出現,
請問這是什麼設置影響造成問題?

圖1
![2023-02-08_095427](https://user-images.githubusercontent.com/5283322/217410332-da945e1e-b8a6-4f86-86e4-08ccf8f1b114.png)
圖2
![2023-02-08_095439](https://user-images.githubusercontent.com/5283322/217410359-014ee4b2-16d3-4d76-9451-3c9954b42748.png)

**Comment by zouyaoji** [comment]
已重现，谢谢反馈，下个版本修复。
